test.js

```diff
@@ -1,22 +1,16 @@
-'use strict';
-var childProcess = require('child_process');
-var test = require('ava');
-var taskkill = require('./');
+import childProcess from 'child_process';
+import test from 'ava';
+import fn from './';
```

Comment:
I think `fn` looks nicer but `m` is probably more "correct".

> But m does not really makes sense of its a function?

Does `m` stand for something specific other than that it's a "module", hah?